apps/api/src/main/resources/data.sql

```diff
@@ -0,0 +1,298 @@
+-- --> Population Script <--
+-- It will populate the following tables WITH sample data:
+-- * roles -> Add the three roles needed by the security service: Admin, User, Developer.
+-- * users -> Add a testing admin user containing a fake email.
+-- * designs -> Add sample designs.
+-- * transaction_items -> Add sample items.
+-- * discounts -> Add some sample discounts.
+-- * newsletter -> Add sample newsletters.
+-- --> Roles TABLE <--
+INSERT INTO `roles`
+SELECT  roles.*
+FROM
+( (
+	SELECT  '358417e5-813b-41b1-baa3-208603253b4c' AS role_id
+	       ,null                                   AS created_at
+	       ,'ROLE_ADMIN'                           AS name
+	       ,null                                   AS updated_at )
+	UNION ALL(
+	SELECT  'c9bc7e4d-e1d3-42de-8ac8-2f83eae57dde'
+	       ,null
+	       ,'ROLE_USER'
+	       ,null )
+	UNION ALL(
+	SELECT  'd3dfa3d5-8000-4096-8985-6b762627ddc1'
+	       ,null
+	       ,'ROLE_DEVELOPER'
+	       ,null )
+) roles
+WHERE NOT EXISTS (
+SELECT  *
+FROM `roles` );
+-- --> End Roles TABLE <--
+-- --> Users TABLE <--
+INSERT INTO `users`
+SELECT  users.*
+FROM
+( (
+	SELECT  'f12d7bcc-ac5f-48f8-8b74-055622e128f0'                         AS user_id
+	       ,null                                                           AS city
+	       ,null                                                           AS country
+	       ,null                                                           AS created_at
+	       ,'admin@not.real'                                               AS email
```

Comment:
The domain should be `example.com`. This ensures that the domain address is valid (and thus the email sending doesn't fail) whilst preventing anyone owning the `not.real` domain from getting the emails
While `not.real` domain doesn't appear to be a registered domain (thus preventing sending emails to someone real), it also can cause email sending to malfunction

This comment also applies to `regular@not.real`